pythonforandroid/bootstrap.py

```diff
@@ -11,6 +13,26 @@
 from pythonforandroid.recipe import Recipe
 
 
+def copy_files(src_root, dest_root, override=True):
+    for root, dirnames, filenames in walk(src_root):
+        for filename in filenames:
+            subdir = root.replace(src_root, "")
+            if subdir.startswith(sep):
+                subdir = subdir[1:]
```

Comment:
Minor suggestion/question:
I wonder if it's not cleaner to use something like `os.path.normpath()`?
```
>>> dest_root = '/foo/'
>>> subdir = 'bar/'
>>> os.path.join(os.path.normpath(dest_root), os.path.normpath(subdir))
'/foo/bar'
```
Which also covers he case were `subdir` by chance ends with two slashes.
So basically we could replace the three lines above by one:
```python
subdir = os.path.normpath(root.replace(src_root, ""))
```

Reply:
normpath won't remove a leading(!) slash, only a trailing one. It could still be used in addition, but I'm pretty certain the point of this code is to make sure `subdir` becomes a relative path, and `normpath` alone wouldn't achieve that